packages/@okta/vuepress-site/docs/guides/oie-embedded-common-run-samples/main/nodejs/runsdkapp.md

```diff
@@ -8,12 +8,20 @@
 1. Open a browser window and navigate to the app's home page: `http://localhost:8080`. After the app loads, click the **Sign In** button located on the home page.
 1. On the sign-in page, enter the username (email) and password you used in [Create your Okta account](/docs/guides/oie-embedded-common-org-setup/nodejs/main/#create-your-okta-account).
 
+   <div class="half">
+
    ![Displays the sample app sign-in form](/img/oie-embedded-sdk/oie-embedded-sdk-sample-app-signin.png)
 
+   </div>
+
 1. Click **Sign In**. If you've successfully signed in, the app redirects you to the user profile page that displays basic user profile and security token information.
 
+   <div class="three-quarter">
```

Comment:
This needs a border else half of it disappears in dark mode.
```suggestion
   <div class="three-quarter border">
```

Reply:
Done in next commit, same comment as above.